### DISCUSSION TITLE
«The Physical Mathematics and Geometry of Dialectical Materialism Versus the Euclidean "Mathematics" and "Geometry" of Philosophical Idealism»

### DISCUSSION THREAD

**Discussion by AngelBlanco19** [a-cosmology-group/acg#253] [Astrophysics and Physics]
This work reveals the antagonistic and unsolvable internal contradictions of the Euclidean "geometry" of Philosophical Idealism with itself and with the mathematics that derives from it, also providing the definitive solution of Dialectical Materialism, which not only solves the aforementioned contradictions, but many others in the field of mathematics, physics, astronomy and cosmology. [https://vixra.org/abs/2402.0116](https://vixra.org/abs/2402.0116)

**Comment by HanDeBruijn** [comment]
As far as Dialectical Materialism is concerned, what do you think of going [Back to the Roots](http://www.alternatievewiskunde.nl/QED/klassiek.htm) ?

**Reply by AngelBlanco19** [reply]
> As far as Dialectical Materialism is concerned, what do you think of going [Back to the Roots](http://www.alternatievewiskunde.nl/QED/klassiek.htm) ?

The question you ask me does not have a short or simple answer, it is extremely long, it would give to write even a book. There are too many lies, deeply rooted in the collective subconscious, especially in the so-called “scientific” community and too many years, too many centuries, several millennia. Several generations educated in totally false paradigms.

If you wish and you find it interesting to see what Dialectical Materialism has to say about it, you can expose this article on this site and we discuss it point by point. I can also do it right here.

Surely you have already understood that Dialectical Materialism is one of the two “parties” that I mention in my introductory part, which I must repeat in every commentary on different topics, and the other “party” is philosophical idealism – everything that is not Dialectical Materialism.

You should also understand that this is not about traditional “Dialectical Materialism", whose dialectical development stalled from 1917 or so, and that I have had to fully develop and deepen to The Causeless Primordial Cause of EVERYTHING, so it is not only about basic philosophy, starting with Parmenides and Zeno, the two greatest geniuses of Humanity in its entire History, but Physics and Mathematics in that order. We will also have to take into account the great contribution made by E. Kant with his «Critique of Pure Reason», where wanting to refute Parmenides, ended up proving everything, even if it seems paradoxical and developed three fundamental concepts for Dialectical Materialism – the concepts of Noumenon, Phenomenon and «the thing-in-itself». Little or none of this can be found in any textbook or alternative source. The road has been long. Knowledge does not fall from the sky.

As you have already seen in this article, which is only the tip of the Iceberg, the Mathematics of Dialectical Materialism emanates from the very essence and structure of Matter itself, it cannot be otherwise. The currently existing ”mathematics“ is the objective cause of Einstein's failure in his General Theory of Relativity to demonstrate the gravitational ”distortion“ of the so-called ”space-time". It is simply impossible within the framework of it because of its falsity. In my other work «The Gravitational Paradox and the Proposed Experiment to Demonstrate Gravitational Distortion of "Spacetime"», I expose in a natural way all this. The so–called “distortion” is the natural state of space - an attributive property of Matter. And Gravity is a physical process, the second most important of Matter and not one of its many side effects, as is currently thought.

You will also have to understand **the radical and fundamental difference** between Reality and the Absolute that creates it. We have access only, solely and exclusively to Reality, never to the Absolute. The “scientific” community naively thinks that it has to do with the Absolute (which is “normal”), when it is only faced with the phenomenological and therefore sensitive and not exact Reality, an infinitesimal part of the Absolute that generates it. The key to everything is given by the only solution of the Aporia «The Arrow» of the great genius of Zeno. If we don't start from here, there is no Science possible. This gave rise to my DOT-DASH-DOT system, which is the Operating System for Calculating Accounting Units of Dialectical Materialism. It is the algorithm that Newton could not see and therefore could not solve, as he himself acknowledges in his famous letters to Bentley.

Again, as you can see, the Mathematics of Dialectical Materialism emanates from the very essence of Matter and not vice versa.

Philosophy, Physics and Mathematics are intrinsically linked in that order, even if their representatives hate each other and do not understand each other. It can't be any other way. It is the only way by which antagonistic, opposite and unsolvable contradictions do not arise. If we really want Science, there is only one way.

**Reply by AngelBlanco19** [reply]
> As far as Dialectical Materialism is concerned, what do you think of going [Back to the Roots](http://www.alternatievewiskunde.nl/QED/klassiek.htm) ?

For Dialectical Materialism to go «Back to the Roots», means to do what I have done, starting with Parmenides and Zeno to get to The Causeless Primordial Cause of EVERYTHING and then, from there, based solely and exclusively on Absolute Truths, without any axioms or postulates and much less with fallacies, to start the algorithm of Dialectical Materialism, which I have developed, to gradually establish a database of Absolute Truths, through the four Laws of Logic, and the now 4 Laws of Dialectical Materialism. It is a very appropriate task for AI, which will “know” EVERYTHING, but will not think, will not reason, will not have Consciousness.

Science, if we want a true Science, cannot be constructed on the basis of axioms, or postulates, which are admitted as an act of Faith, as a belief, without any proof of its veracity. Science can only be built on the basis of Absolute Truths, which must be sought, found, identified, demonstrated, established, recognized and known, in addition to the fact that its intrinsic irrefutability must be demonstrated. This is the reason why I have developed the algorithm of Dialectical Materialism to establish Absolute Truths, which includes its double demonstration – its irrefutability character must be demonstrated. This is only possible if we start from the First Absolute Truth, which is the same as The Causeless Primordial Cause of EVERYTHING.
